### PR TITLE
Support for Falcon2-11B

### PR DESCRIPTION
**Pull request by Nilabhra**
This PR adds support for the Falcon2-11B architecture.

### CODE REVIEW

**Comment by amyeroberts** [comment]
cc @ArthurZucker

File: tests/models/falcon/test_modeling_falcon.py [hunk 1]
```diff
@@ -602,6 +602,25 @@ def test_lm_generate_falcon(self):
 
         self.assertEqual(output_str, EXPECTED_OUTPUT)
 
+    @slow
+    @require_bitsandbytes
+    def test_lm_generate_falcon_11b(self):
```
Comment (ArthurZucker) [inline tests/models/falcon/test_modeling_falcon.py:607]: thanks for adding a test!

**Review by ArthurZucker — APPROVED** [review]
LGTM, let's make the test go green and good to go! 
For outside readers, this does not go against the philosophy as the falcon modeling code is already not very much in accordance to our philosophy 😉 (it has a lot of code paths etc). 
Not super happy but it's the best compromise for this model

**Comment by Nilabhra** [comment]
@ArthurZucker The tests have passed.

**Review by ArthurZucker — APPROVED** [review]
🔥

**Pull request merged**